src/pods/toolbar/components/zoom-button/zoom-in-button.component.tsx

```diff
@@ -4,14 +4,15 @@ import { ZoomIn } from '@/common/components/icons';
 import classes from '@/pods/toolbar/toolbar.pod.module.css';
 
 export const ZoomInButton = () => {
-  const { zoomIn } = useCanvasViewSettingsContext();
+  const { zoomIn, canvasViewSettings } = useCanvasViewSettingsContext();
 
   return (
     <ToolbarButton
       icon={<ZoomIn />}
       label="Zoom In"
       onClick={zoomIn}
       className={classes.button}
+      disabled={canvasViewSettings.zoomFactor < 4.6}
```

Comment:
Move this to a const, so it's easier to find and has more meaning, avoid magic numbers.

```ts
const MINIMUM_ZOOM_FACTOR_ALLOWED = 4.6
```

Reply:
It's fixed